libs/directus/directus-block/vite.config.ts

```diff
@@ -47,5 +47,6 @@ export default defineConfig({
       // External packages that should not be bundled into your library.
       external: externalDeps,
```

Comment:
```suggestion
      external: [...externalDeps, '@okam/stack-ui'],
```

Si on empeche pas de bundle stack-ui il le build au complet et ça crée des conflits server/client